RateLimiter.go

```diff
@@ -0,0 +1,19 @@
+package main
+
+import (
+	"time"
+
+	"github.com/gliderlabs/ssh"
+	"golang.org/x/time/rate"
+)
+
+func RateLimiter(numberOfRequests int, handler ssh.Handler) ssh.Handler {
```

Comment:
also just based on the first look, but it seems like you've rate limited every user? I don't see any way you identify a specific user that we want to rate limit.

Reply:
yes, this limiter is for every user, so I will make the limiter work per user.